index.js

```diff
@@ -303,9 +328,23 @@ const parseArgs = (config = kEmptyObject) => {
         }
       }
 
-      if (ObjectHasOwn(optionConfig, 'multiple')) {
+      const hasMultipleFlag = ObjectHasOwn(optionConfig, 'multiple');
```

Comment:
```suggestion
      const hasMultipleOption = ObjectHasOwn(optionConfig, 'multiple');
```

Light suggestion for consistency with other `optionConfig` validations e.g. `optionType` and `shortOption`